addon/TabTracker.js

```diff
@@ -112,6 +113,16 @@ TabTracker.prototype = {
     }
   },
 
+  handleUndesiredEvent(payload, experimentId) {
+    payload.action = "activity_stream_undesired_event";
```

Comment:
I personally hate those words like "undesired", "bad", "strange", how about "masga"? We could use this to track other app states next, not necessarily be "undesired". My apologies for my OCD, you can just rename this `action` here and leave others alone, as this is the only thing I'd see 😺